bin/node/src/commands.rs

```diff
@@ -122,7 +122,7 @@ pub struct BootstrapChainCmd {
 
 impl BootstrapChainCmd {
     pub fn run(&self) -> Result<(), Error> {
-        let genesis_authorities = self
+        let genesis_authorities: Vec<AuthorityKeys> = self
```

Comment:
NIt: explicit type annotations, can use `::<Vec<_>>` in collect instead

Reply:
Same as before, I got rid of the type annotation completely.